.github/workflows/archive-full-index.yaml

```diff
@@ -0,0 +1,130 @@
+name: Generate the full index archive for Bitnami Charts
+on:
+  push:
+    branches:
+      - index
+env:
+  LAST_INDEX_URL: https://charts.bitnami.com/bitnami/index.yaml
+  INDEXES_DIR: indexes
+jobs:
+  get:
+    runs-on: ubuntu-latest
+    name: Get
+    steps:
+      - id: checkout-repo-index-branch
+        name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          ref: index
+          path: index_branch
+      - id: checkout-repo-full-index-branch
+        name: Checkout repo
+        uses: actions/checkout@v3
```

Comment:
nit-pick: I would make ourselves a favor and simplify the directory names (especially to avoid mixing `-` and `_`)
What about just `index` and `full-index`?

Reply:
heheh naming is hard!, fixed!